packages/kbn-search-connectors/components/sync_jobs/documents_panel.tsx

```diff
@@ -24,21 +25,66 @@ export const SyncJobDocumentsPanel: React.FC<SyncJobDocumentsPanelProps> = (sync
   const columns: Array<EuiBasicTableColumn<SyncJobDocumentsPanelProps>> = [
     {
       field: 'added',
-      name: i18n.translate('searchConnectors.index.syncJobs.documents.added', {
-        defaultMessage: 'Upserted',
-      }),
+      name: (
+        <EuiToolTip
+          content={
+            <FormattedMessage
+              id="searchConnectors.index.syncJobs.documents.upserted.tooltip"
+              defaultMessage="The number of {index} operations the connector sent to the Elasticsearch _bulk API during this sync. This includes net-new documents and updates to existing documents. This does not account for duplicate _ids, or any documents dropped by an ingest processor"
+              values={{ index: <EuiCode>index</EuiCode> }}
+            />
+          }
+        >
+          <>
+            {i18n.translate('searchConnectors.index.syncJobs.documents.added', {
+              defaultMessage: 'Upserted',
+            })}
+            <EuiIcon size="s" type="questionInCircle" color="subdued" className="eui-alignTop" />
+          </>
+        </EuiToolTip>
+      ),
     },
     {
       field: 'removed',
-      name: i18n.translate('searchConnectors.index.syncJobs.documents.removed', {
-        defaultMessage: 'Deleted',
-      }),
+      name: (
+        <EuiToolTip
+          content={
+            <FormattedMessage
+              id="searchConnectors.index.syncJobs.documents.deleted.tooltip"
+              defaultMessage="The number of {delete} operations the connector sent to the Elasticsearch _bulk API at the conclusion of this sync. This may include documents dropped by Sync Rules. This does not include documents dropped by ingest processors. Documents are deleted from the index if the connector determines that they are no longer present in the data that should be fetched from the 3rd-party source."
+              values={{ delete: <EuiCode>delete</EuiCode> }}
+            />
+          }
+        >
+          <>
+            {i18n.translate('searchConnectors.index.syncJobs.documents.removed', {
+              defaultMessage: 'Deleted',
+            })}
+            <EuiIcon size="s" type="questionInCircle" color="subdued" className="eui-alignTop" />
+          </>
+        </EuiToolTip>
+      ),
     },
     {
       field: 'volume',
-      name: i18n.translate('searchConnectors.index.syncJobs.documents.volume', {
-        defaultMessage: 'Volume',
-      }),
+      name: (
+        <EuiToolTip
+          content={
+            <FormattedMessage
+              id="searchConnectors.index.syncJobs.documents.volume.tooltip"
+              defaultMessage="The volume, in MB, of JSON data sent with {index} operations to the Elasticsearch _bulk API during this sync. The current Elasticsearch Index size may be larger, depending on index mappings and settings. It also may be smaller, if large data is substantially trimmed during any ingest processors."
```

Comment:
should be: "trimmed by ingest processors"

Reply:
```suggestion
              defaultMessage="The volume, in MB, of JSON data sent with {index} operations to the Elasticsearch _bulk API during this sync. The current Elasticsearch Index size may be larger, depending on index mappings and settings. It also may be smaller, if large data is substantially trimmed by ingest processors."
```